akvo/rsr/views/my_rsr.py

```diff
@@ -545,7 +545,7 @@ def my_results(request, project_id, template='myrsr/my_results.html'):
     me_managers = project.publishing_orgs.employments().approved().\
         filter(group__in=[admins_group, me_managers_group])
     # Can we unlock and approve?
-    user_is_me_manager = user.is_superuser or user.is_admin or user.me_manager_for_project(project)
+    user_is_me_manager = user.has_perm('rsr.lock_unlock_indicatorperiod')
```

Comment:
Maybe rename this to can_lock_unlock_indicatorperiod? So as not to confuse what this boolean actually represents.

Reply:
I think `can_lock_unlock_indicatorperiod` can still be confusing, because `user_is_me_manager` is used in a bunch of places to differentiate the UI shown for a M&E Manager from that of the Project Editor.  May be the permission name can be changed to something like, `rsr.do_me_manager_actions`?

Reply:
Yup, sounds like a good solution!